app/routes/events/view/export.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return this.modelFor('events.view');
```

Comment:
@niranjan94 I want the event identifier in controller. That's why I am returning it here.